### DISCUSSION TITLE
[Loki] Step should be configurable like in Prometheus data source

### DISCUSSION THREAD

**Discussion by cykl** [grafana/grafana#40851] [Data Sources]
While Prometheus data source allows to set a custom step, Loki data source doesn't are step is always auto-computed by Grafana.

Prometheus:
![image](https://user-images.githubusercontent.com/1764725/138662867-5ed02bcc-b63e-46a0-a7f8-8d6f46974f71.png)

Loki:
![image](https://user-images.githubusercontent.com/1764725/138663075-341de258-31ba-44d1-a94e-1136260b297d.png)

I don't see any reason to this discrepancy and had like to be able to set a custom step. 

As a workaround I was able to use _min interval_ from panel's _query options_ but it works with panel (query inspector shows that step is set to 1800s) but not in explore view.

![image](https://user-images.githubusercontent.com/1764725/138663643-35d9c686-4c49-4c60-a425-0523a963b331.png)

**Comment by ivanahuckova** [comment]
Hello @cykl, this is a duplicate of https://github.com/grafana/grafana/discussions/39022. Would you mind closing this one and sharing your use case for step editor in here https://github.com/grafana/grafana/discussions/39022.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
